std_msgs/Header header
ros_queue_msgs/TransmissionVector target_action
ros_queue_msgs/TransmissionVector applied_action
ros_queue_msgs/TransmissionVector optimal_action

ros_queue_msgs/QueueServerStats queue_server_stats

int8 controller_action_index
int8 applied_action_index
int8 action_index_difference 
int8 optimal_action_index_difference